### PR TITLE
chore: move CLI `exp setup` to `exp setup-opentelemetry`

### PR DESCRIPTION
**Pull request by thedavidprice**
I'm flattening the command structure `exp setup [command]` to improve discoverability. This is part 1 of a few more steps coming, which will make handling experimental packages and setup more feasible.

@Josh-Walker-GM Please do review and let me know your thoughts. NOTE: the Forum post instructions will need updating. I'm happy to do so — just let me know: https://community.redwoodjs.com/t/opentelemetry-support-experimental/4772

### CODE REVIEW

**Comment by Josh-Walker-GM** [comment]
I'll review now. I didn't have any high-level plans when I did the initial setup of the `exp` CLI option. I only tried to make it as top level and so generic as possible. Imagining `exp` could prefix any of the existing cli command types, e.g. `setup` or `generate`. I don't have any strong opinions on structing it a particular way.

File: packages/cli/src/commands/experimental/setupOpentelemetryHandler.js [hunk 1]
```diff
@@ -23,15 +23,15 @@ export const handler = async ({ force, verbose }) => {
...
-    '@prisma/instrumentation',
+    // '@opentelemetry/api',
+    // '@opentelemetry/instrumentation',
+    // '@opentelemetry/exporter-trace-otlp-http',
+    // '@opentelemetry/resources',
+    // '@opentelemetry/sdk-node',
+    // '@opentelemetry/semantic-conventions',
+    // '@opentelemetry/instrumentation-http',
+    // '@opentelemetry/instrumentation-fastify',
+    // '@prisma/instrumentation',
```
[the remark this answers is not in the document]
Comment (thedavidprice) [reply packages/cli/src/commands/experimental/setupOpentelemetryHandler.js:34]: WHOOPS is my thinking on that. Was for debugging purposes only. Too many wheels in motion this morning... thus the review!! Thank you.

Comment (thedavidprice) [inline packages/cli/src/commands/experimental/setupOpentelemetryHandler.js:34]:
```suggestion
    '@opentelemetry/api',
    '@opentelemetry/instrumentation',
    '@opentelemetry/exporter-trace-otlp-http',
    '@opentelemetry/resources',
    '@opentelemetry/sdk-node',
    '@opentelemetry/semantic-conventions',
    '@opentelemetry/instrumentation-http',
    '@opentelemetry/instrumentation-fastify',
    '@prisma/instrumentation',
```

**Comment by thedavidprice** [comment]
> I only tried to make it as top level and so generic as possible. Imagining exp could prefix any of the existing cli command types, e.g. setup or generate. I don't have any strong opinions on structing it a particular way.

Roger that. This is meant to be iterative. And I think we can resolve better overall via a different approach — tbd when we meet this week.

**Review by Josh-Walker-GM — APPROVED** [review]
Okay, I'll update the forum post if you like. Looks to me like the only change needed is to the command name?

**Comment by thedavidprice** [comment]
Correct! And thank you

**Pull request merged**